bin/scriptLib

```diff
@@ -1,4 +1,6 @@
-#! /bin/bash
+#!/usr/bin/env bash
+
+# Copyright (C) 2009-2019 Lightbend Inc. <https://www.lightbend.com>
```

Comment:
Added copyright on all scripts.

Also updated the _hash-bang_ command and now IntelliJ picks this file as bash even if it doesn't have an filename extension.

Reply:
We don't distribute the scripts so it's unnecessary (or, worst, more maintenance if there are follow up changes to the copyright years)